tests/rosetta_test.py

```diff
@@ -226,6 +226,13 @@ def test_flat_field_correction():
     assert corrected_img.shape == input_img.shape
     assert not np.array_equal(corrected_img, input_img)
 
+    # test empty image
+    with pytest.warns(UserWarning, match="Image for flatfield correction is empty"):
+        input_img = np.zeros((10, 10))
+        corrected_img = rosetta.flat_field_correction(img=input_img)
+
+        assert not np.any(input_img)
```

Comment:
You probably meant to check the `corrected_img`, not the `input_img`. Although I suppose they're both equal to zero.

Reply:
Whoops my b, fixed